README.md

```diff
@@ -12,3 +12,29 @@ make it easier for developers to build AI-applications that integrate with Table
 ## Official Documentation
 
 https://tableau.github.io/tableau-mcp/
+
+## Quick Start
+
+### Requirements
+
+- Node.js 20 or newer
+- An MCP client e.g. Claude Desktop, Cursor, VS Code, MCP Inspector, etc.
+
+Standard config works in most MCP clients:
+
+```json
+{
+  "mcpServers": {
+    "tableau": {
+      "command": "npx",
+      "args": ["@tableau/mcp-server@latest"],
+      "env": {
+        "SERVER": "https://my-tableau-server.com",
+        "SITE_NAME": "my_site",
+        "PAT_NAME": "my_pat",
+        "PAT_VALUE": "pat_value"
+      }
+    }
+  }
+}
+```
```

Comment:
I'm adding this to the root README so it shows up on npmjs and users won't have to go digging in the docs site.